engine/src/main/scala/cromwell/webservice/SwaggerUiHttpService.scala

```diff
@@ -94,6 +96,11 @@ trait SwaggerUiConfigHttpService extends SwaggerUiHttpService {
  * swagger UI, but defaults to "yaml". This is an alternative to spray-swagger's SwaggerHttpService.
  */
 trait SwaggerResourceHttpService {
+
+  def getBasePathOverride(): Option[String] ={
+   Option(System.getenv("SWAGGER_BASE_PATH"))
+  }
```

Comment:
```suggestion
  def getBasePathOverride(): Option[String] = {
    Option(System.getenv("SWAGGER_BASE_PATH"))
  }
```
Cleaned up the spacing/indenting a bit.